include/arch/arm/arch/32/mode/hardware.h

```diff
@@ -54,7 +54,7 @@
 
 /* The first physical address to map into the kernel's physical memory
  * window */
-#define PADDR_BASE physBase
+#define PADDR_BASE physBase()
```

Comment:
Can't we drop `PADDR_BASE`then completely and use `physBase()` (or better `get_physBase()`) everywhere. It's very confusing in the C code when thing that look like macro constants turn out to be function. And they can't be used in macros that build values needed at compile time.

Reply:
`get_physBase` does not meet naming seL4 conventions. `getPhysBase` indicates that we are obtaining it from some state, not really a fan of this option

Reply:
I agree, `getPhysBase()` sounds like it is *not* a constant.

Reply:
regarding PADDR_BASE, the concept appears on most (all?) arches, shows up in memory map comments and in verification. I'd be reluctant to mess with that as by-the-way to this PR